### PR TITLE
Heartbeat, Harmony, Dynamic pages

### PR DESCRIPTION
**Pull request by larsfinander**
// 1.1.10
// Dynamic preferences
// Harmony workaround
// Heartbeat zwaveinfo fix
// Setup heartbeat when discovery is called
// Added singleInstance

### CODE REVIEW

**Comment by larsfinander** [comment]
@surfous Have a look

File: src/smartthings/smartapps/smartthings/amazon-echo-home-skill.src/amazon-echo-home-skill.groovy [hunk 1]
```diff
@@ -15,12 +15,23 @@ definition(
 		iconX2Url: "https://s3.amazonaws.com/smartapp-icons/Convenience/App-AmazonEcho@2x.png",
 		iconX3Url: "https://s3.amazonaws.com/smartapp-icons/Convenience/App-AmazonEcho@3x.png",
 		oauth: [displayName: "Amazon Echo (QA)", displayLink: ""]
+		singleInstance: true
```
Comment (surfous) [inline src/smartthings/smartapps/smartthings/amazon-echo-home-skill.src/amazon-echo-home-skill.groovy:18]: @larsfinander  You need a comma at the end of the preceding line ;)
Pending that, I approve ( we need a 99 44/100 emoji )

**Comment by surfous** [comment]
@larsfinander Left a line comment in the file itself. I can't fix it myself, but take care of that tiny issue then I approve.

**Comment by surfous** [comment]
👍

**Pull request merged**